content/docs/rfcs/38/README.md

```diff
@@ -0,0 +1,21 @@
+---
+slug: 38
+title: 38/LOGOS-CONSENSUS
+name: Initial Logos Consensus Description
+status: incomplete-pre-draft
```

Comment:
```suggestion
status: raw
```

Reply:
I'd stick to making the status one of `raw`, `draft`, `stable`.
In this case `raw` is what this PR works towards.